Jenkinsfile

```diff
@@ -329,6 +329,8 @@ pipeline {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
                                    ignored_files: "src/control/vendor/*:src/include/daos/*.pb-c.h:src/common/*.pb-c.[ch]:src/mgmt/*.pb-c.[ch]:src/iosrv/*.pb-c.[ch]:src/security/*.pb-c.[ch]:*.crt:*.pem:*_test.go:src/cart/_structures_from_macros_.h"
+                        sh label: "codespell",
+			   script: "codespell --ignore-words-list nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable --builtin clear --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden `git ls-tree --full-tree --name-only HEAD`"
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'ths' may be misspelled - perhaps 'the'?
(style)  'ba' may be misspelled - perhaps 'by'?
(style)  'creat' may be misspelled - perhaps 'create'?
(style)  'te' may be misspelled - perhaps 'the'?
(style)  'mapp' may be misspelled - perhaps 'map'?
(style)  'dout' may be misspelled - perhaps 'doubt'?
(style)  'tre' may be misspelled - perhaps 'tree'?
(style)  'reord' may be misspelled - perhaps 'record'?
(style)  'mimick' may be misspelled - perhaps 'mimic'?
(style)  'cloneable' may be misspelled - perhaps 'clonable'?